kythe/go/indexer/emit.go

```diff
@@ -933,6 +944,22 @@ func (e *emitter) writeRef(origin ast.Node, target *spb.VName, kind string) *spb
 		} else {
 			e.writeEdge(target, rule.VName, rule.EdgeOut)
 		}
+
+		if e.opts.emitDerivedLinkageFileGeneratesEdges() {
+			if rule.VName.Path != "" && target.Path != "" {
+				var ruleVName = rule.VName
```

Comment:
This doesn't copy the VName.  We're mutating the underlying rule.

Reply:
Fixed.